internal/signing/channelaccounts/channel_accounts_model.go

```diff
@@ -31,6 +31,7 @@ type ChannelAccount struct {
 type ChannelAccountStore interface {
 	GetIdleChannelAccount(ctx context.Context, lockedUntil time.Duration) (*ChannelAccount, error)
 	Get(ctx context.Context, sqlExec db.SQLExecuter, publicKey string) (*ChannelAccount, error)
+	GetAllByPublicKey(ctx context.Context, sqlExec db.SQLExecuter, publicKeys ...string) ([]*ChannelAccount, error)
```

Comment:
Do we still need the `Get` function in this case?

Reply:
Yes, we do want to preserve compatibility with the TSS, which uses this method.